managers/asset-operations-manager.js

```diff
@@ -972,11 +978,15 @@ class AssetOperationsManager {
         this.validationService.validateAssetUpdateCancel(UAL, blockchain);
 
         const { tokenId } = resolveUAL(UAL);
-        await this.blockchainService.cancelAssetUpdate(tokenId, blockchain);
+        let receipt = await this.blockchainService.cancelAssetUpdate(tokenId, blockchain);
 
         return {
             UAL,
-            operation: getOperationStatusObject({ status: 'COMPLETED' }, null),
+            operation: {
+                status: getOperationStatusObject({ status: 'COMPLETED' }, null),
```

Comment:
This on is not needed, since we only have 1 operation in this function, I would just return `operation: receipt`